R-package/R/lgb.importance.R

```diff
@@ -44,22 +43,33 @@ lgb.importance <- function(model, percentage = TRUE) {
   tree_dt <- lgb.model.dt.tree(model)
 
   # Extract elements
-  tree_imp <- tree_dt %>%
-    magrittr::extract(.,
-                      i = ! is.na(split_index),
-                      j = .(Gain = sum(split_gain), Cover = sum(internal_count), Frequency = .N),
-                      by = "split_feature") %T>%
-    data.table::setnames(., old = "split_feature", new = "Feature") %>%
-    magrittr::extract(., i = order(Gain, decreasing = TRUE))
+  tree_imp_dt <- tree_dt[
```

Comment:
comma should at beginning of lines or at end of lines by convention?

I think we should standardize them in the package, likely to be at beginning of lines as it's easier/clearer to read.

Reply:
I believe it should be beginning of lines for R code. Would you like me to open an issue and make a PR to standardize?

Reply:
opened #2373 and added PR #2437 to address it